streams/src/main/java/org/apache/kafka/streams/errors/StreamsNotStartedException.java

```diff
@@ -17,11 +17,12 @@
 package org.apache.kafka.streams.errors;
 
 import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.KafkaStreams.State;
 
 /**
- * Indicates that Kafka Streams is in state {@link KafkaStreams.State#CREATED CREATED} and thus state stores cannot be queries yet.
+ * Indicates that Kafka Streams is in state {@link State CREATED} and thus state stores cannot be queries yet.
```

Comment:
We can only link to the enum, but not its values.

Reply:
Ah!